src/pages/AppDetails/AppInfo/AppDescription.tsx

```diff
@@ -0,0 +1,124 @@
+import * as React from 'react';
+import { Row, Col, ListView, ListViewItem, ListViewIcon, Icon } from 'patternfly-react';
+import PfInfoCard from '../../../components/Pf/PfInfoCard';
+import { DisplayMode, HealthIndicator } from '../../../components/Health/HealthIndicator';
+import { AppHealth } from '../../../types/Health';
+import { App, AppWorkload } from '../../../types/App';
+import { WorkloadIcons } from '../../../types/Workload';
+import { IstioLogo } from '../../../config';
+import { Link } from 'react-router-dom';
+
+type AppDescriptionProps = {
+  app: App;
+  health?: AppHealth;
+};
+
+type AppDescriptionState = {};
+
+class AppDescription extends React.Component<AppDescriptionProps, AppDescriptionState> {
+  constructor(props: AppDescriptionProps) {
+    super(props);
+    this.state = {};
+  }
+
+  istioSidecar() {
+    let istioSidecar = true;
+    this.props.app.workloads.forEach(wkd => {
+      istioSidecar = istioSidecar && wkd.istioSidecar;
+    });
+    return istioSidecar;
+  }
+
+  serviceLink(namespace: string, service: string) {
+    return '/namespaces/' + namespace + '/services/' + service;
+  }
+
+  workloadLink(namespace: string, workload: string) {
+    return '/namespaces/' + namespace + '/workloads/' + workload;
+  }
+
+  renderServices(namespace: string, workload: string, serviceNames: string[]) {
+    let iconType = 'pf';
+    let iconName = 'service';
+    return serviceNames.map(service => (
+      <div key={'workload_' + workload + '_service_' + service} className="ServiceList-Title">
+        <Icon type={iconType} name={iconName} className="service-icon" />
+        <Link to={this.serviceLink(namespace, service)}>{service}</Link>
+      </div>
+    ));
+  }
+
+  renderWorkloadItem(namespace: string, workload: AppWorkload) {
+    /*
+      Not sure if we need a common icon per Workload instead of an icon per type of Workload
+     */
+    let iconName = WorkloadIcons.Deployment;
```

Comment:
In my opinion, it is good to have an icon for each type :) It gives more information to the user, it is more explicit. And this should help to detect problems easier.

Reply:
Yes, inside the "Workloads" page.
I mean, I know that we are under workloads, then a different icon per workload type (deployment/replicaset/etc.).
But in this page, I was not sure, It sounded weird to me to set a different icon per workload without bring the "workload details".
That was my "rationale" for this comment.
I guess I forgot to mention that context was related just to this page, not in general.